Source/Scene/CreditDisplay.js

```diff
@@ -376,10 +399,18 @@ CreditDisplay.prototype.addCredit = function (credit) {
     return;
   }
 
+  let credits;
   if (!credit.showOnScreen) {
-    this._currentFrameCredits.lightboxCredits.set(credit.id, credit);
+    credits = this._currentFrameCredits.lightboxCredits;
+  } else {
+    credits = this._currentFrameCredits.screenCredits;
+  }
+
+  if (credits.contains(credit.id)) {
+    const creditCount = credits.get(credit.id).count;
+    credits.set(credit.id, new CreditDisplayElement(credit, creditCount + 1));
```

Comment:
Instead of creating a new CreditDisplayElement, could the `count` of the existing CreditDisplayElement be incremented?